plugin/copilot_chat.vim

```diff
@@ -28,7 +28,7 @@ augroup CopilotChat
   autocmd!
   autocmd FileType copilot_chat autocmd BufDelete <buffer> call copilot_chat#buffer#on_delete(expand('<abuf>'))
   autocmd FileType copilot_chat autocmd BufEnter,TextChanged,TextChangedI <buffer> call copilot_chat#buffer#apply_code_block_syntax()
-  if has('patch-9.0.0917')
+  if has('patch-9.0.0917') || has('nvim')
```

Comment:
👍

Reply:
@sodapopcan I resolved the conflicts. Can we merge this PR?

Reply:
Ha, while I've been quite chatty in this repo I'm not an owner.  @DanBradbury will have to do it.

Also, despite my thumbs up, it would be clearer to just do `if exists('#WinResized')` and the patch+nvim check.  Not sure why I didn't just do that in the first place.